nautobot/circuits/models.py

```diff
@@ -339,8 +345,9 @@ def to_objectchange(self, action):
             changed_object=self,
             object_repr=str(self),
             action=action,
-            related_object=related_object,
             object_data=serialize_object(self),
+            object_datav2=object_datav2,
+            related_object=related_object,
         )
```

Comment:
Let's:

1. Create a utility helper function for constructing the object_datav2 field, similar to what `serialize_object` does for the existing object_data.
2. Make it so that `ChangeLoggedModel.to_objectchange()` takes appropriate appropriate optional arguments so that subclasses can just call `super().to_objectchange(...)` instead of needing to have repeated code in each case.

In other words:

```python
class ChangeLoggedModel(models.Model):
    ...
    def to_objectchange(self, action, related_object=None, object_data_extra=None, object_data_exclude=None):
        return Objectchange(
           changed_object=self,
           object_repr=str(self),
           action=action,
           object_data=serialize_object(self, extra=object_data_extra, exclude=object_data_exclude),
           object_data_v2=serialize_object_v2(self),
        )
```

```python
class CircuitTermination(...):
    ...

    def to_objectchange(self, action, **kwargs):
        try:
            kwargs["related_object"] = self.circuit
        except Circuit.DoesNotExist:
            # Parent circuit has been deleted
            kwargs["related_object"] = None
        return super().to_objectchange(action, **kwargs)
```

```python
class RackGroup(...):
    ...
    def to_objectchange(self, action, **kwargs):
        kwargs["object_data_exclude"] = ["level", "lft", "rght", "tree_id"]
        return super().to_objectchange(action, **kwargs)
```

etc.